modules/imgproc/src/hough.cpp

```diff
@@ -105,51 +105,69 @@ array of (rho, theta) pairs. linesMax is the buffer size (number of pairs).
 Functions return the actual number of found lines.
 */
 static void
-HoughLinesStandard( const Mat& img, float rho, float theta,
-                    int threshold, std::vector<Vec2f>& lines, int linesMax,
+HoughLinesStandard( InputArray image, OutputArray lines,
+                    float rho, float theta,
+                    int threshold, int linesMax,
                     double min_theta, double max_theta )
 {
+    int type = CV_32FC2;
+    if( lines.fixedType() )
+    {
+        type = lines.type();
+        CV_Assert( type == CV_32FC2 || type == CV_32FC3 );
+    }
+
+    Mat img = image.getMat();
+
     int i, j;
     float irho = 1 / rho;
 
     CV_Assert( img.type() == CV_8UC1 );
 
-    const uchar* image = img.ptr();
+    const uchar* _image = img.ptr();
```

Comment:
I changed the name of this variable because I changed the argument of this function to "InputArray image". However, it should not modifiy existing code as much as possible. I change the argument name to "InputArray src" and return this variable name to its original name.